extension/chrome/elements/pgp_pubkey.ts

```diff
@@ -98,7 +101,12 @@ Catch.try(async () => {
       for (const pubkey of pubs) {
         const email = Str.parseEmail(pubkey.users[0].userId ? pubkey.users[0].userId!.userid : '').email;
         if (email) {
-          contacts.push(await Store.dbContactObj({ email, client: 'pgp', pubkey: pubkey.armor(), lastUse: Date.now(), lastSig: await Pgp.key.lastSig(pubkey) }));
+          contacts.push(await Store.dbContactObj(
+            {
+              email, client: 'pgp', pubkey: pubkey.armor(), lastUse: Date.now(), lastSig: await Pgp.key.lastSig(pubkey),
+              expiresOn: Number(await Pgp.key.dateBeforeExpiration(pubkey)) || undefined
```

Comment:
`Number(await Pgp.key.dateBeforeExpiration(pubkey)) || undefined`
If this needs to be always done, we could (should) move it inside `dbContactObj`. So that here we can just do `expiresOn: await Pgp.key.dateBeforeExpiration(pubkey)`